tests/cucumber/features/step_definitions/organizations_steps.coffee

```diff
@@ -40,7 +40,7 @@ do ->
 
     @Then /^I should be on the "([^"]*)" detail page$/, (name, callback) ->
       @browser
-        .waitForVisible('.organization-detail', assert.ifError)
+        .waitForVisible('.organization-detail', 30000, assert.ifError)
```

Comment:
I wouldn't want that in individual tests but maybe we could change the default wait time based on it.